parsers/CA_NS.py

```diff
@@ -37,21 +37,37 @@ def fetch_production(country_code='CA-NS', session=None):
     }
     """
     r = session or requests.session()
-    url = 'http://www.nspower.ca/system_report/today/currentmix.json'
-    response = r.get(url)
+    mix_url = 'http://www.nspower.ca/system_report/today/currentmix.json'
+    mix_data = r.get(mix_url).json()
+
+    load_url = 'http://www.nspower.ca/system_report/today/currentload.json'
+    load_data = r.get(load_url).json()
+
     data = []
-    for obj in response.json():
+    for mix in mix_data:
+        corresponding_load = [load_period for load_period in load_data
+                              if load_period['datetime'] == mix['datetime']]
+
+        # in mix_data, the values are expressed as percentages,
+        # e.g. "Solid Fuel":  45.76 meaning 45.76%
+        # Divide load by 100.0 to simplify calculations later on.
```

Comment:
Actually that's a good point. http://www.nspower.ca/en/home/about-us/how-we-make-electricity/imported-electricity.aspx the only exchange is with NB (and to Quebec/Maine through there). The problem is the API for NS only tells us when there are some imports to Nova Scotia, not when they have exports. Right now, the NS API says imports are 0, NB says they are importing 56 MW from NS. So that's not going to be an exact number.

But the other thing is this parser currently takes imports as "unknown". I think we could just remove that, and depend on NB API to know when NS is exporting from NB.

I will take a look over next few hours/days to see if the values reported by NS imports and NB exports line up when NS is importing.